voting_schemes/electionguard/verifier/src/services/bulletin-board-client.service.js

```diff
@@ -0,0 +1,62 @@
+import ApolloClient from "apollo-client";
+import { createHttpLink } from "apollo-link-http";
+import fetch from "node-fetch";
+import InMemoryCache from "apollo-cache-inmemory";
+import gql from "graphql-tag";
+
+/**
+ * This class includes some methods to perform requests to the Bulletin Board
+ * server GraphQL API.
+ * @class
+ */
+export class BulletinBoardClient {
```

Comment:
At some point we could think of using the BB npm package, once it'll be ready to be used with node 🚀